README.md

```diff
@@ -36,63 +37,66 @@ Fostering discussions about how to prevent overfitting may be an outcome of the
 Using data from earlier date that 2020/01/01 is also allowed during the prediction phase.
 The forecast start dates in this dataset are from 2020/01/02 to 2020/12/31.
-  - For all 3 models : 
-    - forecast_time : from 2020/01/02 to 2020/12/31, weekly every 7 days (every Thurday).
-    - valid_time (forecast_time + lead_time): from 2020/01/02 to 2020/12/31
+  - For all 3 models: 
+    - `forecast_time`: from 2020/01/02 to 2020/12/31, weekly every 7 days (every Thurday).
+    - `valid_time` (`forecast_time` + `lead_time`): from 2020/01/02 to 2021/02/xx <!-- valid_time extends into 2021 -->
   - ECMWF forecast
-    - lead_time : 0 to 46 days
+    - `lead_time`: 0 to 46 days
     - available parameters (same as for Hindcast input (training input)
   - ECCC forecast 
-    - lead_time : 1 to 32 days
+    - `lead_time`: 1 to 32 days
     - available parameters (same as for Hindcast input (training input)
   - NCEP forecast 
-    - lead_time : 1 to 44 days
+    - `lead_time`: 1 to 44 days
     - available parameters (same as for Hindcast input (training input)
 
  List of files :
   [grib](https://storage.ecmwf.europeanweather.cloud/s2s-ai-challenge/data/test-input/0.3.0/grib/index.html),
   [netcdf](https://storage.ecmwf.europeanweather.cloud/s2s-ai-challenge/data/test-input/0.3.0/netcdf/index.html),
-  zarr
+  zarr (missing)
   
 ### Observations (Reference Output)
 The `hindcast-like-observations` (`training-output-reference`) dataset.
 The `forecast-like-observations` (`test-output-reference`) dataset.
 
-The observations are the ground truth to compare with the ML model output and evaluate them. It consists in observation from instruments of temperature and accumulated total precipitation. (TODO add more descriptions) (point to the scripts to create them ? TODO).
-Generally speaking, only past data can be used by the ML models to perform their forecast :
+The observations are the ground truth to compare with the ML model output and evaluate them. It consists in observation from instruments of [temperature](http://iridl.ldeo.columbia.edu/SOURCES/.NOAA/.NCEP/.CPC/.temperature/.daily/) and accumulated total [precipitation](http://iridl.ldeo.columbia.edu/SOURCES/.NOAA/.NCEP/.CPC/.UNIFIED_PRCP/.GAUGE_BASED/.GLOBAL/.v1p0/.extREALTIME/.rain/). The [NOAA CPC](https://www.cpc.ncep.noaa.gov/) datasets were downloaded from [IRIDL](iridl.ldeo.columbia.edu/). We provide observations in the same dimensions as the forecasts/hindcasts to have an easy match of forecasts/hindcast and ground truth. [See the script for technical details](https://github.com/ecmwf-lab/climetlab-s2s-ai-challenge/tree/main/tools/observations).
+
+
+Generally speaking, only data available when the forecast is issued can be used by the ML models to perform their forecast:
 
 __Rule 1 : Observed data beyond the forecast date should not be used for prediction, for instance a forecast starting on 2020/07/01 should not use observed data beyond 2020/07/01).__
```

Comment:
this sounds like there are also other rules here? I think you had here more in the past, but I see this now as the only rule.